starboard/raspi/shared/open_max/video_decoder.cc

```diff
@@ -66,8 +66,9 @@ void VideoDecoder::Initialize(const DecoderStatusCB& decoder_status_cb,
   error_cb_ = error_cb;
 
   SB_DCHECK_EQ(thread_, 0);
-  pthread_create(&thread_, nullptr, &VideoDecoder::ThreadEntryPoint, this);
-  SB_DCHECK_NE(thread_, 0);
```

Comment:
There are other comparisons and `DCHECK`s between `thread_` and 0 in the destructor and right above these changes.